lib/spack/spack/build_systems/oneapi.py

```diff
@@ -9,16 +9,26 @@
 import shutil
 from os.path import basename, isdir
 
-from llnl.util.filesystem import HeaderList, find_libraries, join_path, mkdirp
+from llnl.util import tty
+from llnl.util.filesystem import HeaderList, LibraryList, find_libraries, join_path, mkdirp
 from llnl.util.link_tree import LinkTree
 
+from spack.build_environment import dso_suffix
 from spack.directives import conflicts, variant
+from spack.package_base import InstallError
 from spack.util.environment import EnvironmentModifications
 from spack.util.executable import Executable
 
 from .generic import Package
 
 
+def raise_lib_error(*args):
+    """Bails out with an error message. Shows args after the first as one per
+    line, tab-indented, useful for long paths to line up and stand out.
+    """
+    raise InstallError("\n\t".join(str(i) for i in args))
```

Comment:
As far as I can tell you only ever pass one string to this function so consider using `raise InstallError` directly?

Reply:
I had copied it over from intel.py because it was used by the openmp code. You are right, it is overkill for its use in oneapi.py. I agree with the other comments. I expect to add openmp support to intel-oneapi-dal in the future. I set automerge so it already merged with the approval. I can make the changes after your PR is done.